src/web/app/components/sortable-table/sortable-table.component.ts

```diff
@@ -11,6 +11,22 @@ export interface ColumnData {
   sortBy?: SortBy; // optional if the column is not sortable
 }
 
+/**
+ * Data provided for each table cell
+ * Priority of display
+ * 1. customComponent
+ * 2. displayValue
+ * 3. value
+ */
+export interface CustomTableCellData {
+  value?: any; // Optional value used for sorting with sortBy provided in ColumnData
```

Comment:
The value should not be optional?

Reply:
Some custom components are not sortable, e.g. last column of `contribution-question-statistic`. Should only need to input `customComponent` without `value`.